mdbackup/storage/s3.py

```diff
@@ -72,27 +71,25 @@ def upload(self, path: Path, parent: Union[Path, str, str] = None):
             key = path.name
         key = self.__ok_key(key)
         self.__log.info(f'Uploading file {key} (from {path})')
-        ret = self.__s3.upload_file(str(path.absolute()),
-                                    self.__bucket,
-                                    key,
-                                    ExtraArgs={
-                                        'ContentType': magic.from_file(str(path.absolute()), mime=True),
-                                        'StorageClass': self.__storageclass,
-                                    })
-        self.__log.debug(ret)
+        self.__s3.upload_file(str(path.absolute()),
+                              self.__bucket,
+                              key,
+                              ExtraArgs={
+                                  'ContentType': magic.from_file(str(path.absolute()), mime=True),
+                                  'StorageClass': self.__storageclass,
+                              })
 
     def delete(self, path: Union[Path, str]):
-        path = self.__ok_key(path)
-        self.__log.info(f'Deleting {self.__pre}/{path}')
+        full_path = self.__ok_key(path)
```

Comment:
This fixes the delete method which created a wrong path and tried to remove it: `backups/pi/backups/pi/sth` now will be `backups/pi/sth`